ora_mktest.sql

```diff
@@ -147,4 +155,51 @@ INSERT INTO tab3 (id, tab2_id, f, ids)
 
 COMMIT;
 
+/* connect as "testschema3" to create some partitioned tables */
+CONNECT testschema3/good_password
+
+CREATE TABLE part1 (c1 integer, c2 varchar2(100))
+  PARTITION BY LIST (c1) (
+    PARTITION part1_a VALUES (1, 2, 3),
+    PARTITION part1_b VALUES (4, 5),
+    PARTITION part1_default VALUES (DEFAULT)
+  );
+
+CREATE TABLE part2 (c1 integer, c2 varchar2(100))
+  PARTITION BY RANGE (c1) (
+    PARTITION part2_a VALUES LESS THAN (0),
+    PARTITION part2_b VALUES LESS THAN (100),
+    PARTITION part2_c VALUES LESS THAN (MAXVALUE)
+  );
+
+CREATE TABLE part3 (c1 integer, c2 varchar2(100))
+  PARTITION BY HASH (c1) (
+    PARTITION part3_a,
+    PARTITION part3_b,
+    PARTITION part3_c
+  );
+
+CREATE TABLE part4 (c1 char(1), c2 date)
+  PARTITION BY LIST (c1)
+  SUBPARTITION BY RANGE (c2) (
+    PARTITION part4_a VALUES ('A') (
+      SUBPARTITION part4_a_2020 VALUES LESS THAN 
+        (TO_DATE('01-JAN-2021','dd-MON-yyyy')),
+      SUBPARTITION part4_a_2021 VALUES LESS THAN 
+        (TO_DATE('01-JAN-2022','dd-MON-yyyy')),
+      SUBPARTITION part4_a_2022 VALUES LESS THAN 
+        (TO_DATE('01-JAN-2023','dd-MON-yyyy'))
+    ),
+    PARTITION part4_b VALUES ('B') (
+      SUBPARTITION part4_b_2020 VALUES LESS THAN 
+        (TO_DATE('01-JAN-2021','dd-MON-yyyy')),
+      SUBPARTITION part4_b_2021 VALUES LESS THAN 
+        (TO_DATE('01-JAN-2022','dd-MON-yyyy')),
+      SUBPARTITION part4_b_2022 VALUES LESS THAN 
+        (TO_DATE('01-JAN-2023','dd-MON-yyyy'))
+    )
+  );
+
+COMMIT;
```

Comment:
Thanks that you thought of regression tests!

Could you insert a few rows so that we see if the data get migrated?

Reply:
Done! https://github.com/cybertec-postgresql/ora_migrator/pull/42/commits/73edf4734c4a281911ecf9a285e458e3cd122088